pennylane/fermi/conversion.py

```diff
@@ -299,3 +300,263 @@ def _(fermi_operator: FermiSentence, n, ps=False, wire_map=None, tol=None):
         return qubit_operator.map_wires(wire_map)
 
     return qubit_operator
+
+
+def bravyi_kitaev(
+    fermi_operator: Union[FermiWord, FermiSentence],
+    n: int,
+    ps: bool = False,
+    wire_map: dict = None,
+    tol: float = None,
+) -> Union[Operator, PauliSentence]:
+    r"""Convert a fermionic operator to a qubit operator using the Bravyi-Kitaev mapping.
+
+    .. note::
+
+        Hamiltonians created with this mapping should be used with operators and states that are
+        compatible with the Bravyi-Kitaev basis.
```

Comment:
We don't need to resolve this now, but it would be great for us to link, e.g., to a demo to explain what we mean here.

Reply:
We will explain this in the planned [demo](https://app.shortcut.com/xanaduai/epic/59264?group_by=none&vc_group_by=day&ct_workflow=all&cf_workflow=500000005).